src/CodeGen_Internal.cpp

```diff
@@ -388,6 +388,42 @@ Expr lower_int_uint_mod(const Expr &a, const Expr &b) {
     }
 }
 
+Expr unsigned_long_div(Expr num, const Expr &den, const Scope<Interval> &bounds) {
+    internal_assert(num.type() == den.type());
+    internal_assert(num.type().is_uint());
+    Type ty = num.type();
+    Expr leading_zeros, q;
+    q = make_zero(ty);
```

Comment:
Style nit: better to declare-and-init in the same statements, eg

```
    Expr q = make_zero(ty);
    Expr leading_zeros = cast(ty, count_leading_zeros(den));
```

Reply:
Done